homework5.py

```diff
@@ -0,0 +1,37 @@
+# 1 Task
+print('Task 1: \"Palindrome\"')
+my_str = input("Enter string for check: ")
+if (my_str == my_str[::-1]):
+    first_check = True
+    print(f'The statement that this string is palindrome is {first_check}')
+else:
+    second_check = False
+    print(f'The statement that this string is palindrome is {second_check}')
```

Comment:
print(f'The statement that this string is palindrome is {my_str == my_str[::-1]}') - this enough for solve this task. You don't need if else block